benchmarks/collections/hash-dict-vs-blob-dict_get.ts

```diff
@@ -0,0 +1,56 @@
+import { pathToFileURL } from "node:url";
+import { add, complete, configure, cycle, save, suite } from "@typeberry/benchmark/setup.js";
+import { Bytes } from "@typeberry/bytes";
+import { StringHashDictionary } from "@typeberry/collections";
+import { BlobDictionary } from "@typeberry/collections/blob-dictionary.js";
+import { HASH_SIZE, type OpaqueHash } from "@typeberry/hash";
+
+const NO_OF_KEYS = 1000;
+
+function key(n: number) {
+  const key = Bytes.blobFromString(`${n}`);
+  const ret = Bytes.zero(HASH_SIZE);
+  ret.raw.set(key.raw, 0);
+  return ret;
```

Comment:
my suggestion would be to create two kinds of benchmarks:
1. One that is using keys with collisions on long prefixes.
2. Second one that hashes the keys (i.e. evenly distributes the prefixes).

Reply:
done